satellite/operations/operations.py

```diff
@@ -97,10 +91,12 @@ def evaluate(self, flow: HTTPFlow, phase: Phase):
         ))
 
         if output_message:
-            if input_message.body != output_message.body:
-                flow_phase_obj.text = output_message.body
             if input_message.headers != output_message.headers:
-                flow_phase_obj.headers = Headers(output_message.headers)
+                flow_phase_obj.headers = \
+                    Headers([(name.encode('UTF-8'), value.encode('UTF-8'))
+                             for name, value in output_message.headers.items()])
```

Comment:
Just a style suggestion
```suggestion
                flow_phase_obj.headers = Headers([
                    (name.encode('UTF-8'), value.encode('UTF-8'))
                    for name, value in output_message.headers.items()
                ])
```